src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -150,17 +162,14 @@ public String getApiUri() {
                 return;
             }
         } else {
-            listener.getLogger().format("Connecting to %s with no credentials, anonymous access%n", apiUri == null ? "github.com" : apiUri);
+            listener.getLogger().format("Connecting to %s with no credentials, anonymous access%n", apiUri == null ? GITHUB_URL : apiUri);
         }
 
         GHOrganization org = null;
         try {
             org = github.getOrganization(repoOwner);
         } catch (RateLimitExceededException rle) {
-            listener.getLogger().format("%n%s%n%n", rle.getMessage());
-            throw new InterruptedException();
-        } catch (IOException e) {
-            // may be a user... ok to ignore
```

Comment:
So what happens now when it *is* a user? Was this logic wrong all along? Have you retested all the configuration combinations?